src/utils/addressHelper.js

```diff
@@ -1,16 +1,5 @@
 export const buildInlineAddressFromObject = (address) => {
-  let inlineAddress = "";
-  const propsToUse = ["line1", "line2", "postalCode"];
-
-  for (let i = 0; i < propsToUse.length; i++) {
-    let tempProp = address[propsToUse[i]];
-    if (tempProp != null) {
-      inlineAddress += tempProp;
-      if (i < propsToUse.length - 1) {
-        inlineAddress += " ";
-      }
-    }
-  }
+  const inlineAddress = `${address.line1}${address.line2 ? ` ${address.line2}` : ""}, ${address.city}, ${address.regionCode}, ${address.postalCode}, ${address.country}`;
```

Comment:
what if some of those props are not defined? there will be undefined in final string. not what we want.